cocos/2d/CCNode.h

```diff
@@ -38,6 +38,10 @@
 #include "2d/CCComponentContainer.h"
 #include "2d/CCComponent.h"
 
+#ifdef CC_STUDIO_ENABLED_VIEW   // for cocostudio only
+#include "CocosStudioExtension.h"
+#endif
+
```

Comment:
I don't think it is a good idea to do it like this because other files that don't need this header file will also include it too. It will slow down the compiling speed. We should include as less header file as possible.